targets/mbedos5/tools/generate_pins.py

```diff
@@ -0,0 +1,226 @@
+"""
+getpins_js.py
```

Comment:
Name is incorrect. Moreover, I suggest to let this file start just like other python files in the project:
```py
#!/usr/bin/env python

# Copyright 2016 Samsung Electronics Co., Ltd.
# Copyright 2016 University of Szeged.
#
# Licensed under the Apache License, Version 2.0 (the "License");
# you may not use this file except in compliance with the License.
# You may obtain a copy of the License at
#
#     http://www.apache.org/licenses/LICENSE-2.0
#
# Unless required by applicable law or agreed to in writing, software
# distributed under the License is distributed on an "AS IS" BASIS
# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
# See the License for the specific language governing permissions and
# limitations under the License.
```

That is,
- no repetition of the file name, so you wont have to keep it in sync with actual file name,
- let first line be a hashbang line,
- put license in comment, not in docstring.